docs/release_notes/v1.1.0.md

```diff
@@ -0,0 +1,380 @@
+          - docs.spiceai.org
+          - huggingface.co
+  ```
+
+  For more details, see the [Perplexity documentation](https://spiceai.org/docs/components/models/perplexity).
+
+- **Web Search Tool**: The new Web Search Tool enables Spice models to search the web for information using search engines like Perplexity. Example configuration:
+
+  ```yaml
+  tools:
+    - name: the_internet
+      from: websearch
+      description: 'Search the web for information.'
+      params:
+        engine: perplexity
+        perplexity_auth_token: ${ secrets:SPICE_PERPLEXITY_AUTH_TOKEN }
+  ```
+
+  For more details, see the [Web Search Tool documentation](https://spiceai.org/docs/components/tools/websearch).
+
+- **Eval Scorers**: Eval scorers assess model performance on evaluation cases. Spice includes built-in scorers:
+
+  - `match`: Exact match.
+  - `json_match`: JSON equivalence.
+  - `includes`: Checks if actual output includes expected output.
+  - `fuzzy_match`: Normalized subset matching.
+  - `levenshtein`: Levenshtein distance.
+
+  Custom scorers can use embedding models or LLMs as judges. Example:
+
+  ```yaml
+  evals:
+    - name: australia
+      dataset: cricket_questions
+      scorers:
+        - hf_minilm
+        - judge
+        - match
+  embeddings:
+    - name: hf_minilm
+      from: huggingface:huggingface.co/sentence-transformers/all-MiniLM-L6-v2
+  models:
+    - name: judge
+      from: openai:gpt-4o
+      params:
+        openai_api_key: ${ secrets:OPENAI_API_KEY }
+        system_prompt: |
+          Compare these stories and score their similarity (0.0 to 1.0).
+          Story A: {{ .actual }}
+          Story B: {{ .ideal }}
+  ```
+
+  For more details, see the [Eval Scorers documentation](https://spiceai.org/docs/features/large-language-models/evals#Eval-Scorers).
+
+- **IMAP Data Connector**: Query emails stored in IMAP servers using federated SQL. Example:
+
+  ```yaml
+  datasets:
+    - from: imap:myawesomeemail@gmail.com
+      name: emails
+      params:
+        imap_access_token: ${secrets:IMAP_ACCESS_TOKEN}
+  ```
+
+  For more details, see the [IMAP Data Connector documentation](https://spiceai.org/docs/components/data-connectors/imap).
+
+- **Automatic NSQL Query Retries**: Failed NSQL queries are now automatically retried, improving reliability for federated queries. For more details, see the [NSQL documentation](https://spiceai.org/docs/reference/nsql).
+
+- **Enhanced Task Tracing**: Task history now includes chat completion IDs, and runtime readiness is traced for better observability. Use the `runtime.task_history` table to query task details. See the [Task History documentation](https://spiceai.org/docs/reference/task_history).
+
+- **Graceful Shutdown for HTTP Server**: The HTTP server now supports a configurable request drain for graceful shutdowns, ensuring smoother runtime termination. See the [HTTP Server documentation](https://spiceai.org/docs/components/http-server).
```

Comment:
```suggestion
- **Graceful Shutdown for HTTP Server**: The HTTP server now supports a request drain for graceful shutdowns, ensuring smoother runtime termination. See the [HTTP Server documentation](https://spiceai.org/docs/components/http-server).
```

We don't currently have shutdown timeout configurable. I can't also find [HTTP Server documentation](https://spiceai.org/docs/components/http-server)